projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/helpers/google-drive.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, __experimentalHStack as HStack } from '@wordpress/components'; // eslint-disable-line @wordpress/no-unsafe-wp-apis
+import { __ } from '@wordpress/i18n';
+import GoogleSheetsIcon from '../../../../../icons/google-sheets';
+import GoogleDriveConnectButton from '../components/GoogleDriveConnectButton';
+import GoogleDriveDisconnectButton from '../components/GoogleDriveDisconnectButton';
+import type { CardItem, CardBuilderProps } from './types';
+
+export function buildGoogleDriveCard( {
+	integration,
+	refreshIntegrations,
+	context,
+	handlers,
+}: CardBuilderProps ): CardItem {
+	const isConnected = !! integration.isConnected;
+	const settingsUrl = integration.settingsUrl as string | undefined;
+
+	const defaultResponsesUrl = '/wp-admin/admin.php?page=jetpack-forms-admin';
+	const responsesUrl =
+		( window as unknown as { jpFormsBlocks?: { defaults?: { formsResponsesUrl?: string } } } )
+			.jpFormsBlocks?.defaults?.formsResponsesUrl || defaultResponsesUrl;
```

Comment:
Can this be replaced with the new useConfig thingy? Also thinking that in CIAB-admin we'll need different URLs? 🤔

Reply:
We don't currently supply this via use config, but we probably could. And removing this kind of thing is exactly why we introduced the config endpoint. I'll explore that in a follow up.